backend/src-common/src/main/java/com/siemens/sw360/components/summary/ReleaseSummary.java

```diff
@@ -36,33 +37,41 @@ public ReleaseSummary(VendorRepository vendorRepository) {
     @Override
     protected Release summary(SummaryType type, Release document) {
         Release copy = new Release();
-        copyField(document, copy, _Fields.ID);
-        copyField(document, copy, _Fields.NAME);
-        copyField(document, copy, _Fields.VERSION);
-        copyField(document, copy, _Fields.COMPONENT_ID);
-        copyField(document, copy, _Fields.CLEARING_TEAM_TO_FOSSOLOGY_STATUS);
-        copyField(document, copy, _Fields.FOSSOLOGY_ID);
+        if(type == SummaryType.DETAILED_EXPORT_SUMMARY){
+           setDetailedExportSummaryFields(document, copy);
+        } else {
+            copyField(document, copy, _Fields.ID);
+            copyField(document, copy, _Fields.NAME);
+            copyField(document, copy, _Fields.VERSION);
+            copyField(document, copy, _Fields.COMPONENT_ID);
+            copyField(document, copy, _Fields.CLEARING_TEAM_TO_FOSSOLOGY_STATUS);
+            copyField(document, copy, _Fields.FOSSOLOGY_ID);
 
-        if(document.isSetVendorId() ) {
+            if (type != SummaryType.SHORT) {
+                copyField(document, copy, _Fields.CPEID);
+                copyField(document, copy, _Fields.CREATED_BY);
+                copyField(document, copy, _Fields.MAINLINE_STATE);
+                copyField(document, copy, _Fields.CLEARING_STATE);
+                copyField(document, copy, _Fields.RELEASE_DATE);
+                copyField(document, copy, _Fields.LANGUAGES);
+                copyField(document, copy, _Fields.OPERATING_SYSTEMS);
+                copyField(document, copy, _Fields.ATTACHMENTS);
+                copyField(document, copy, _Fields.MAIN_LICENSE_IDS);
+            }
+        }
+        if (document.isSetVendorId()) {
             final String vendorId = document.getVendorId();
             if (!Strings.isNullOrEmpty(vendorId)) {
                 copy.setVendor(vendorRepository.get(vendorId));
             }
         }
+        return copy;
+    }
 
-        if (type != SummaryType.SHORT) {
-            copyField(document, copy, _Fields.CPEID);
-            copyField(document, copy, _Fields.CREATED_BY);
-            copyField(document, copy, _Fields.MAINLINE_STATE);
-            copyField(document, copy, _Fields.CLEARING_STATE);
-            copyField(document, copy, _Fields.RELEASE_DATE);
-            copyField(document, copy, _Fields.LANGUAGES);
-            copyField(document, copy, _Fields.OPERATING_SYSTEMS);
-            copyField(document, copy, _Fields.ATTACHMENTS);
-            copyField(document, copy, _Fields.MAIN_LICENSE_IDS);
+    private void setDetailedExportSummaryFields(Release document, Release copy) {
+        for (_Fields renderedField : ReleaseExporter.RENDERED_FIELDS) {
```

Comment:
That' s not so easy, as only the ReleaseExporter knows the rendered fields. And actually even the ReleaseExporter knows only which fields should not be exported and writes the rest to the RENDERED_FIELDS. The list depends therefore on the fields that are currently defined for releases.